scikits/umfpack/setup.py

```diff
@@ -28,6 +28,13 @@ def umfpack_i(ext, build_dir):
     build_info = {}
     dict_append(build_info, **umf_info)
     dict_append(build_info, **blas_info)
+    
+    # On windows, vcpkg names the library 'libumfpack'
+    # rather than 'umfpack' and this change is not yet
+    # in numpy; therefore, we fix the problem here
+    if sys.platform == 'win32':
+        build_info['libraries'].remove('umfpack')
+        build_info['libraries'].append('libumfpack')
```

Comment:
Does not this break non-vcpkg based builds? This change was not required when I tried (successfully at last) to build with the m2w64 tool chain. Unfortunately I cannot test this right now.

Reply:
The `sys.platform=='win32'` should indicate that the official Python.org Python (compiled with MSVC) is running.

Reply:
Thanks for explanation.

Reply:
I want so say that just because something *should* be the case doesn't mean that it is. You should probably just check it yourself.